nbs/18_callback.fp16.ipynb

```diff
@@ -202,18 +202,49 @@
     "    def after_loss(self): self.autocast.__exit__(None, None, None)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        "Tests whether fp16 gradients have overflowed."
</pre><p>Could be <code>Tests whether fp16 gradients have overflow.</code>?  or maybe <code>are overflowed</code>  (really dont know as I fail on english here).</p><p> </p><p>I Also see some docstrings with an ending <code>.</code>  and others without it, it seems that all docstrings should end with a dot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
Ah, "overflown" is probably the correct word.